### ISSUE TITLE
quantile error testing example

### ISSUE THREAD

**Issue by hagenw** [TWOEARS/Auditory-Machine-Learning-Training-and-Testing-Pipeline#1]
I was testing the [Train sound type identification models](http://twoears.aipa.tu-berlin.de/doc/latest/examples/train-identification/) example, and got the following error:

Go to the folder `train_identification_model` in TWOEARS/examples and run

```
>> trainAndTestCleanModel

[...]

Running: MultiConfigurationsEarSignalProc
==========================================
./home/hagen/git/twoears/twoears-data/sound_databases/IEEE_AASP/pageturn/pageturn19.wav
..............................
::Undefined function or variable 'quantile'.

Error in dataProcs.SceneEarSignalProc.runningEnergy (line 192)
            energy = energy - quantile(energy,0.98);


Error in dataProcs.SceneEarSignalProc/makeEarsignalsAndLabels (line 106)
                [energy, tFramesSec] = dataProcs.SceneEarSignalProc.runningEnergy( obj.getDataFs(), double(obj.earSout(:,ii)), 100e-3, 50e-3 );


Error in dataProcs.SceneEarSignalProc/process (line 35)
            obj.makeEarsignalsAndLabels( inputFileName );


Error in dataProcs.MultiConfigurationsEarSignalProc/makeEarsignalsAndLabels (line 72)
                    obj.sceneProc.process( wavFileName );


Error in dataProcs.MultiConfigurationsEarSignalProc/process (line 41)
            obj.makeEarsignalsAndLabels( inputFileName );


Error in core.DataPipeProc/run (line 95)
                    obj.dataFileProcessor.process( inputFileName );


Error in core.IdentificationTrainingPipeline/run (line 140)
                obj.dataPipeProcs{ii}.run();


Error in trainAndTestCleanModel (line 38)
modelPath = pipe.pipeline.run( {classname}, 0 );
```

**Comment by ivo--t** [comment]
fixed by 25f27f95599f06dabeb89c68b4f3c6ab283ca0b0. Please verify and close :)

**Comment by hagenw** [comment]
Works again, thanks.
